tests/common.py

```diff
@@ -19,7 +19,7 @@
 
 TESTS_HOME = Path(__file__).parent
```

Comment:
Another case of `pathlib` usage.

Reply:
Perhaps adding the `pathlib` library would make it so that you don't need to cast paths to strings?

Reply:
The issue is that Python 2.7 does not `open` Path objects. To be clear, the pathlib library seems to be installed, even though it's not explicitly in the environment file, otherwise there would be an ImportError when I import pathlib. 

Since we're going to move to Py3 where pathlib is built-in, I suggest we use `str` for now and remove them when we drop Py2 support.